manualTests/src/main/x/TestMisc.x

```diff
@@ -89,70 +89,56 @@ module TestMisc {
         console.print("~b=" + ~b);
     }
 
-    void testChars() {
-        console.print("\n** testChars()");
-
+    void testAsciiDigit() {
+        console.print("\n** testAsciiDigit()");    
         Char[] chars = "1aA!\n$£€".toCharArray();
-        for (Char ch : chars) {
-            String dec = "";
-            if (Int value := ch.decimalValue()) {
-                dec = $"\'{value}\'";
-            }
-
-            console.print($|char {ch.toSourceString()}, unicode={ch.unicode}, cat={ch.category},\
-                             | lower={ch.lowercase.toSourceString()}, upper={ch.uppercase.toSourceString()},\
-                             | title={ch.titlecase.toSourceString()}, dec={dec}, num={ch.numericValue}
-                             );
-        }
-
         // this also tests the conditional UInt8 to Int conversion
         assert Int n := chars[0].asciiDigit(), n == 1;
         assert !chars[1].asciiDigit();
     }
 
-    void testStrings() {
-        console.print("\n** testStrings()");
-        assert "abcde"[2] == 'c';
-
-        assert "abcde".substring(0) == "abcde";
-        assert "abcde".substring(-5) == "abcde";
-        assert "abcde".substring(-99) == "abcde";
-        assert "abcde".substring(2) == "cde";
-        assert "abcde".substring(-3) == "cde";
-        assert "abcde".substring(4) == "e";
-        assert "abcde".substring(-1) == "e";
-        assert "abcde".substring(5) == "";
-        assert "abcde".substring(6) == "";
-        assert "abcde".substring(99) == "";
-
-        assert "abcde"[1..2] == "bc";
-        assert "abcde"[2..1] == "cb";
-        assert "abcde"[1..<2] == "b";
-        assert "abcde"[2..<1] == "c";
-        assert "abcde"[1>..<2] == "";
-        assert "abcde"[2>..<1] == "";
-        assert "abcde"[1>..2] == "c";
-        assert "abcde"[2>..1] == "b";
-    }
-
-    void testCast() {
-        console.print("\n** testCast()");
-
-        Int    i = 42;
-        Object o = i;
-        console.print("o=" + o);
-        Int    n = o.as(Int);
-        console.print("n=" + n);
-
-        Object o2 = Int:4;
-        console.print("o2=" + o2);
-
-        try {
-            console.print("i.as(String) should throw " + i.as(String));
-        } catch (Exception e) {
-            console.print("i.as(String) failed correctly: \"" + e.text + '"');
-        }
-    }
+    // Requires unicode character handling
```

Comment:
Since this test logic is part of master, should it be merged in commented out? This will reduce the test code coverage for existing backend.